### PR TITLE
Update dbcc-checkdb-transact-sql.md

### PR DESCRIPTION
**Pull request by mlaskiMSFT**
Added detail outlining that in order to Perform logical consistency checks on indexes, the database must be in a readable state.  "It order for the internal query to succeed the database must be in a readable state."

### CODE REVIEW

**Comment by Court72** [comment]
@rwestMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
@Boogieman2599 Can you please provide more clarity? What does "readable" mean? Do you mean "online"? Also, please note the typo "It". I think you meant "In.

**Comment by mlaskiMSFT** [comment]
hi @rwestMSFT thanks for reviewing this. The readable means that it should be set to at least read only access. e.g. this command would not work if the database was set to a non-readable secondary replica. The database would by default need to be online so that it is readable. If a database is offline this would make it unreadable for any queries.  Yes, thanks for correcting my typo in the summary but I think I corrected this in the main body of the proposed text: it should read "It order for the internal query to succeed the database must be in a readable state". I hope this helps to clarify the change. Please reach out to me if anything further is required.

File: docs/t-sql/database-console-commands/dbcc-checkdb-transact-sql.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ Logical consistency checking on indexes varies according to the compatibility le
...
+These logical consistency checks cross check the internal index table of the index object with the user table that it is referencing. To find outlying rows, an internal query is constructed to perform a full intersection of the internal and user tables. It order for the internal query to succeed the database must be in a readable state. Running this query can have a significant effect on performance, and its progress can't be tracked.  Therefore, we recommend that you specify `WITH EXTENDED_LOGICAL_CHECKS` only if you suspect index issues that are unrelated to physical corruption, or if page-level checksums have been turned off and you suspect column-level hardware corruption.
```
Comment (rwestMSFT) [inline docs/t-sql/database-console-commands/dbcc-checkdb-transact-sql.md]:
```suggestion
These logical consistency checks cross check the internal index table of the index object with the user table that it is referencing. To find outlying rows, an internal query is constructed to perform a full intersection of the internal and user tables. In order for the internal query to succeed, the database must be in a readable state. Running this query can have a significant effect on performance, and its progress can't be tracked. Therefore, we recommend that you specify `WITH EXTENDED_LOGICAL_CHECKS` only if you suspect index issues that are unrelated to physical corruption, or if page-level checksums have been turned off and you suspect column-level hardware corruption.
```

**Comment by rwestMSFT** [comment]
@Boogieman2599 I've reached out to someone from the product group to confirm your suggestion, which may require additional wording changes. Thank you for the clarification.

**Comment by rwestMSFT** [comment]
On the advice of PG, we're closing this until an IcM can be raised.

@mlaskiMSFT In future, changes to content from internal folks should be made in `sql-docs-pr`, not here. Thanks for your understanding.

**Pull request closed without merging**